[pt] Localize otel-localized announcement page to Portuguese (Brazil)

pedroespindula opened the pull request:
Waiting for #4996 to be merged.

Add localization for annoucement page for OTel localization.

Issue: https://github.com/open-telemetry/opentelemetry.io/issues/4922 and https://github.com/open-telemetry/opentelemetry.io/issues/4863

lgfa29 commented on content/pt/blog/2024/docs-localized.md:
```diff
@@ -0,0 +1,38 @@
+
+Suas contribuições podem gerar um impacto significativo!
+
+Nos agradecemos a todos os contribuidores por fazer todo o possível, e nos
+estamos ansiosos para ver como essas novas opções de linguagens podem melhorar a
+experiência do usuário do OpenTelemetry.
```
```suggestion
Nós agradecemos todas as pessoas que contribuíram para tornar isso uma realidade, e
estamos ansiosos para ver como essas novas opções de idioma podem melhorar a
experiência de usuário do OpenTelemetry.
```

"make this possible" é tipo "tornar realidade", não necessariamente fazer o possível.

lgfa29 reviewed (commented):
Ótimo trabalho @pedroespindula!

edsoncelio commented on content/pt/blog/2024/docs-localized.md:
```diff
@@ -0,0 +1,38 @@
+default_lang_commit: 48eac183a4dd74946d5a45fa436cfc6052f30532
+---
+
+Estamos felizes em anunciar que o site do OpenTelemetry está disponível em
```
+1 pra segunda sugestão!

EzzioMoreira replied:
Voto pela primeira sugestão.

edsoncelio commented on content/pt/blog/2024/docs-localized.md:
```diff
@@ -0,0 +1,38 @@
+O projeto OpenTelemetry cresceu para incluir contribuidores e usuários de todo o
+mundo. Criar um site disponível em múltiplas línguas é um passo importante para
+garantir que todos, independente da sua língua nativa, consigam contribuir para
+o projeto. Nos também estamos esperando que essa iniciativa conceda aos usuários
+finais acesso a documentações em diversas linguagens, facilitando o aprendizado
```
Acho que não chegamos a um consenso pra "end user", ótima pergunta.

Nesse contexto acho que se refere mais a usuário final mesmo.

EzzioMoreira replied:
Foi traduzido literalmente: usuário final. 
https://github.com/open-telemetry/opentelemetry.io/pull/4997/files#diff-e25eb4d4a8d751fe6eba2e26ad39ea46656a4063f5f6e5f2e9de9d84afb74009R131

edsoncelio commented on content/pt/blog/2024/docs-localized.md:
```diff
@@ -0,0 +1,38 @@
+Convidamos você para contribuir nesse trabalho. Você pode encontrar colegas que
+contribuem para localização no canal do Slack
+[#otel-docs-localization](https://cloud-native.slack.com/archives/C076RUAGP37).
+Se você é fluente em uma linguagem que já é suportada, você pode ajudar criando
```
"idiomas suportados" também parece bom!

edsoncelio commented on content/pt/blog/2024/docs-localized.md:
```diff
@@ -0,0 +1,38 @@
+Se você é fluente em uma linguagem que já é suportada, você pode ajudar criando
+ou revisando traduções. Se sua língua ainda não é suportada e você gostaria de
+ajudar a traduzir,
+[crie uma issue](<https://github.com/open-telemetry/opentelemetry.io/issues/new?title=Add+%3CYOUR%20LANGUAGE%3E+(%3CYOUR+CODE%3E)+version+of+website+pages&body=%3C!--+Provide+github+handles+of+at+least+2+people+that+will+work+on+this+translation+project%20--%3E>).
```
Também não temos tradução pra issue ainda.
Não tenho muito uma opinião sobre traduzir já que é um nome comum do ecossistema do GitHub.

pedroespindula replied:
Concordo com o @edsoncelio para esse caso, nesse sentido, irei optar por não traduzir o termo.

theletterf commented:
@pedroespindula The announcement should go live today ideally. Could you have a look at the feedback please?

chalin reviewed (changes requested):
Hi. This PR will need to be **rebased**, have conflicts resolved, and then force-pushed.

pedroespindula commented:
> @pedroespindula The announcement should go live today ideally. Could you have a look at the feedback please?

> Hi. This PR will need to be **rebased**, have conflicts resolved, and then force-pushed.

Hey, @theletterf and @chalin! 

Thanks for the heads up! I have updated the PR according to the feedbacks and *rebased* the PR. Also, I made the PR *Ready for Review*. 

Also, thanks for the review @lgfa29!

Cc: @edsoncelio

edsoncelio reviewed (commented):
LGTM!!

EzzioMoreira commented:
LGTM 🆗 
Parabéns pela iniciativa @pedroespindula 👏

theletterf commented:
/fix:format

chalin commented:
@open-telemetry/docs-pt-approvers - please (GH) approve

chalin reviewed (approved):
LGTM
Awaiting official approval from @open-telemetry/docs-pt-approvers before merging.

edsoncelio commented:
@emdneto 
![image](https://github.com/user-attachments/assets/5de26c4d-5624-4f16-ba7f-fc46a7dd3a5d)

chalin commented:
@open-telemetry/docs-maintainers - any idea why @open-telemetry/docs-pt-approvers / maintainers isn't a choice in the Reviewers request dropdown?

> <img width="296" alt="image" src="https://github.com/user-attachments/assets/35793ee4-a249-485f-bd66-4f8935ba6d53">

chalin commented:
Merging on @emdneto's approval.
